build.gradle.kts

```diff
@@ -45,4 +45,8 @@ allprojects {
 
 tasks.withType<org.jetbrains.kotlin.gradle.targets.js.npm.tasks.KotlinNpmInstallTask>().configureEach {
     args.add("--ignore-engines")
-}
\ No newline at end of file
+}
+
+dependencyLocking {
+    lockAllConfigurations()
+}
```

Comment:
Dependency locking and verification were activated, but kept in lenient mode for now, to account for native dependency shift in CI.